release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -581,6 +581,12 @@ With {product-title} 4.12, a new interface has been added to the `nodeip-configu
 
 For more information, see xref:../support/troubleshooting/troubleshooting-network-issues.html#overriding-default-node-ip-selection-logic_troubleshooting-network-issues[Optional: Overriding the default node IP selection logic]. 
 
+[id="new-network-observability"]
+==== New Network Observability Operator to observe network traffic flow
+As an administrator, you can now install Network Observability Operator to observe the network traffic for {product-title} cluster in the console. You can view and monitor the network traffic data in different graphical representations. The Network Observability uses eBPF technology to create the network flows. The network flows are enriched with {product-title} information, and stored in Loki. As an administrator, you can use the network traffic information for detailed troubleshooting and analysis.
```

Comment:
```suggestion
As an administrator, you can now install the Network Observability Operator to observe the network traffic for {product-title} cluster in the console. You can view and monitor the network traffic data in different graphical representations. The Network Observability Operator uses eBPF technology to create the network flows. The network flows are enriched with {product-title} information, and stored in Loki. You can use the network traffic information for detailed troubleshooting and analysis.
```

Reply:
A couple of minor things, adding "the" and "Operator". I'm thinking, delete that 2nd "As an administrator".

Reply:
Yes, agree on removing the 'As an administrator..'